.github/workflows/main.yml

```diff
@@ -0,0 +1,14 @@
+name: Label PRs
+
+on:
+  - pull_request
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+	
```

Comment:
is the indentation ok here?

Reply:
yaml flexible with indentation, but i'll adjust to two spaces